cmd/tempo-cli/main.go

```diff
@@ -44,6 +44,10 @@ var cli struct {
 		Index viewIndexCmd `cmd:"" help:"View contents of block index"`
 	} `cmd:""`
 
+	Gen struct {
+		GenIndex genIndexCmd `cmd:"" help:"Generate bloom/index for a block"`
```

Comment:
Let's rename this to just `Index` to remove the repetition of the command line.  Then it will be `tempo-cli gen index`